### ISSUE TITLE
Recipe show crash

### ISSUE THREAD

**Issue by zortag** [Chicken-Bones/NotEnoughItems#149]
CodeChickenCore-1.7.10-1.0.4.29-universal
NotEnoughItems-1.7.10-1.0.4.83-universal
industrialcraft-2-2.2.659-experimental
HydCraft-1.7.10-2.0.78-universal

i got this crash while whatching any recipe.
http://pastebin.com/Y1eR8Zxy

what is strange is that the crashlog talk about hydraulic craft
i was in creative+ mode and i selected (mouse over) ic2 advanced machine case then i pressed "r" to see the recipe.
then i tried to check other blocks and everytime i get the crash

**Comment by GirafiStudios** [comment]
Yeah this is not a NEI bug. It's a Hydraulicraft bug. Just update Hydraulicraft to the latest version (HydCraft-1.7.10-2.0.84)
